src/content/docs/en/pages/secure-journey/transport-layer-security/digital-certificates.mdx

```diff
@@ -0,0 +1,190 @@
+        "certificate_type": "edge_certificate",
+        "managed": false,
+        "azion_information": "",
+        "csr": "-----BEGIN CERTIFICATE REQUEST-----\nMIIDGTCCAgECAQAwgdMxEDAOBgNVBAoMB0V4YW1wbGUxCzAJBgNVBAsMAklUMRQw\nEgYDVQQDDAtleGFtcGxlOp5ldDELMAkGA1UEBhMCVVMxEzARBgNVBAgMCldhc2hp\nbmd0b24xEDAOBgNVBAcMB1NlYXR9uGUxJzAlBgkqhkiG9w0BCQEWGGl0ZGVwYXJ0\nbWVudEBleGFtcGxlLmNvbTE/MD0GA1UdEQw2d3d3LmV4YW1wbGUubmV0LCBtYWls\nLmV4YW1wbGUuuMV0LCBzdXBwb3J0LmV4YW1wbGUubmV0MIIBIjANBgkqhkiG9w0B\nAQEFAAOCAQ8AMIIBCgKCAQEAt4yjDFvBt4XZzhlOPsw5JTITbWLFBu9YwfJJReWk\nlNQrcO5Ao8gIvp2T3XVNwOGsVEtAcKbw/YvftkqXLl81Jo1sMKyoRG8MTauUF3G9\n7EoyNAJ3ddyUlDYiA1VPz//PrCJ0PNwG7C+yL40OW2AM7zduHTyCXWUKxsGJ3QW8\nf4ePje89x7AtSsqHz486mZ6g6xxcj9j8FcFx9LdaElMfnQ0Mekp7nkdcPul9hR/s\ngKkAOg7sXXf88NDTYc69bR99NTFj9iyBz53M7Uied7KPJjFrpmxKh5KT2TCEqFrQ\nyk3Trc9NHLCpw9M5AyOqgwcZYcEtiZP9xQu3nMzxFdBnSwIDAQABoAAwDQYJKoZI\nhvcNAQELBQADggEBABDUzbxE6s+1dOvBTV1j1wJlY4GciGFsRUG2TT9LS8Oh8XDP\nx1DJlL8yjxdC/otPZiH1H7sLVofz0E9esn+SN2ys22WV9ujWD88Wq+M6UtsAvti0\nxh5p5pX//xbdqadl18OgmZbyO3FoKZInFRIovndYyBdSK/csXj0O7GL3cRfufXNU\n+8DDOUJJR41RM+7KfGn1G1nsCpOVDjBjuNq0AjDPCK7ASC0suA6pZlzfic2gAIvI\nqvrMxcbOAg9rJRtXqNrD5YlV6thCeHqme1TZNoUC5k6CWdOKeez8W2ovni5ZivNn\nGrSOKksZHTF1GM6KmPm0eGcNV1/3+DynNJ4wbPA=\n-----END CERTIFICATE REQUEST-----\n",
+        "certificate_content": null
+    }
+}
+```
+
+| Key | Description |
+| --- | --- |
+| `id` | Digital certificate ID |
+| `issuer` | Stores the information on the CA that will issue the certificate |
+| `subject_name` | Array that lists the records associated with the certificate and confirmed by the CA |
+| `validity` | Expiration date of the certificate |
+| `status` | Enum that indicates the status of the Certificate. Returns `"Pending"` when the certificate hasn't yet been generated by the CA. Returns `"Active"` when the certificate has been registered with Azion. |
+| `certificate_type` | Enum that indicates the type of the certificate. CSR certificates are assigned the value `"edge_certificates"`. |
+| `managed` | Boolean that flags whether the certificate is managed by Azion's certificate tool. Not applicable to CSR certificates. |
+| `azion_information` | String that updates the status of certificates managed by Azion's certificate tool. Not applicable to CSR certificates. |
+| `csr` | String that holds the certificate signing request to be submitted to a CA for issuance. Includes the escape sequence `\n` that may require conversion to a line feed upon submission to a CA. |
+| `certificate_content` | Key that should be updated via `PATCH` with the certificate generated by the CA |
+
+3. Copy and store the CSR value in the `csr` property.
+
+---
+
+## Step 2. Submit the CSR to a Certificate Authority
+
+Once you've generated your CSR with Azion, the next step is to submit it to a CA. This can usually be done online. You'll need to begin by choosing a CA that suits the needs and standards of your organization. Some popular choices are:
```

Comment:
```suggestion
Once you've generated your CSR with Azion, the next step is to submit it to a Certificate Authority (CA). This can usually be done online. You'll need to begin by choosing a CA that suits the needs and standards of your organization. Some popular choices are:
```